datasets/th/designated_person/crawler.py

```diff
@@ -0,0 +1,77 @@
+import re
+from typing import Generator, Dict, Tuple, Optional
+from lxml.etree import _Element
+
+from zavod import Context, helpers as h
+
+
+def parse_table(
+    table: _Element,
+) -> Dict[str, str]:
+    return {
+        row.findtext(".//th"): row.findtext(".//td") for row in table.findall(".//tr")
+    }
+
+
+def crawl_item(url: str, context: Context):
+    response = context.fetch_html(url)
+
+    info_dict = parse_table(response.find(".//table"))
+
+    en_name = info_dict.pop("Individual/Entity Name (English)").strip()
+    th_name = info_dict.pop("Individual/Entity Name (Thailand)").strip()
+
+    entity = context.make("Person")
+    entity.id = context.make_id(en_name)
+
+    entity.add("name", en_name, lang="en")
+    entity.add("name", th_name, lang="th")
+    entity.add("topics", "crime")
```

Comment:
I think this is enough of a standard financial sanction to use the sanction topic.
```suggestion
    entity.add("topics", "sanction")
```